.github/workflows/release.yml

```diff
@@ -95,6 +101,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
+      packages: read
```

Comment:
Necessary here?

Reply:
you have to have packages:read set to pull from the sitemesh github repo

Reply:
Yes, but this job does not pull any packages